usermods/smartnest/usermod_smartnest.h

```diff
@@ -9,6 +9,10 @@
 class Smartnest : public Usermod
 {
 private:
+  bool initialized = false;
+  unsigned long lastMqttReport = 0;
+  unsigned long mqttReportInterval = 60000; // Report every minute
```

Comment:
make this static const (or constexpr is better)

Reply:
I would argue to allow it to be just `unsigned` and use it as a run-time settable variable.

Reply:
So I better leave it like this?

Reply:
Better to add configuration item like other usermods.